Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -380,8 +380,8 @@ ModelExperimental.prototype.destroy = function () {
  * The model can be a traditional glTF asset with a .gltf extension or a Binary glTF using the .glb extension.
  *
  * @param {Object} options Object with the following properties:
- * @param {String|Resource|Uint8Array} [options.gltf] A Resource/URL to a glTF/glb file or a binary glTF buffer
- * @param {Object} [options.basePath=''] The base path that paths in the glTF JSON are relative to.
+ * @param {String|Resource|Uint8Array|Object} [options.gltf] A Resource/URL to a glTF/glb file, a binary glTF buffer, or a JSON object containing the glTF contents
```

Comment:
The property is not optional. Must have overlooked that from a previous review.

```suggestion
 * @param {String|Resource|Uint8Array|Object} options.gltf A Resource/URL to a glTF/glb file, a binary glTF buffer, or a JSON object containing the glTF contents
```